RealmSwift/Tests/ResultsTests.swift

```diff
@@ -186,7 +186,7 @@ class ResultsTests: TestCase {
         innerArray.array.append(SwiftObject())
         outerArray.array.append(innerArray)
         realm.add(outerArray)
-        XCTAssertEqual(Int(1), outerArray.array.filter("ANY array IN %@", realm.objects(SwiftObject)).count)
+        // FIXME: XCTAssertEqual(Int(1), outerArray.array.filter("ANY array IN %@", realm.objects(SwiftObject)).count)
```

Comment:
This test still fails, so @alazier's CVarArgType fix didn't work :disappointed:.

I'd rather not have to force users to do `realm.objects(SwiftObject).map { $0 }`...